.openshift-ci/get_latest_helm_chart_versions.py

```diff
@@ -42,6 +42,9 @@ def main(argv):
     helm_versions = get_latest_helm_chart_versions("stackrox-secured-cluster-services", n)
     logging.info(f"Helm chart versions for the latest {n} releases:")
     print("\n".join(helm_versions))
+    specific_release=Release(major=3, minor=74)
```

Comment:
```suggestion
    specific_release = Release(major=3, minor=74)
```

Reply:
Also this could be defined above the main function with a reference to the support exception.

Reply:
Sure, I'll do that